pages/replays.vue

```diff
@@ -103,18 +106,35 @@
       SimpleFooter,
     },
 
-    data() {
-      return {
-        blurNSFW: true,
-      };
-    },
+    methods: {
+      ...mapMutations ({
+        setBlurNsfw : VStore.$mutations.setBlurNsfw,
+      }),
 
-    methods: {},
+      ...mapActions ({
+        loadSettings : VStore.$actions.loadSettings,
+      }),
+    },
 
     computed: {
+      ...mapState({
+        getBlurNsfw : VStore.$states.blurNsfw,
+      }),
+
+      blurNSFW: {
+        get () { return this.getBlurNsfw; },
+        set ( data ) {
+          this.setBlurNsfw( data );
+        }
+      },
+
       version () {
         return `v${process.env.version}`;
       },
     },
+
+    mounted() {
+      this.loadSettings();
```

Comment:
There may be a better place to trigger load settings. I think in a client side only plugin maybe.